### PR TITLE
feat: [DEV-12506]: Static Import/Export

### PR DESCRIPTION
**Pull request by Sung96kim**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist:

- [ ] Jira story/task is put into PR title
- [ ] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ ] My changes generate no new warnings
- [ ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: indico/queries/model_export.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+    def __init__(self, model_id: int):
+        self.model_id = model_id
+        super().__init__(self.query, variables={"modelId": model_id})
+
+    def process_response(self, response) -> ModelExport:
+        return ModelExport(**super().process_response(response)["createModelExport"])
+
+
+class CreateModelExport(RequestChain):
```
Comment (goatrocks) [inline indico/queries/model_export.py:27]: Can you add argument comments as well? 🙏

File: indico/types/model_export.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import datetime
+
+from indico.types.base import BaseType
+
+
+class ModelExport(BaseType):
```
Comment (goatrocks) [inline indico/types/model_export.py:6]: and comments here too please

**Review by goatrocks — COMMENTED** [review]
just need some doc comments so we can generate readme docs 🥳
